docs/api-docs/apps/guide/apps-03-developing.md

```diff
@@ -82,7 +82,7 @@ Once you've exposed an app to the internet, you can register a draft app in [Dev
    * **Uninstall**: `https://4022ffe4.ngrok.io/uninstall`
 6.  Click **Update & Close**.
 
-7.  Click **View Client ID** to view the app's `client_id` and `client_secret`.

```

Comment:
Are we sure `client_id` is going to be removed from here and will not be used for anything?